.vale/styles/RedHat/Definitions.yml

```diff
@@ -48,6 +48,7 @@ exceptions:
   - DRL
   - DSL
   - EIP
+  - ELB
```

Comment:
Please add ELB term to the testvalid fixture for the Definitions rule.

Reply:
Just this one and then we are good to merge.

Reply:
Added

Reply:
No - add it to `.vale/fixtures/RedHat/Definitions/testvalid.adoc`

There is a 1:1 correspondence between Rules:Fixtures

Reply:
Thanks . I added "ELB" to the testvalid.adoc file in TermErrors and Definitions.